inc/db.function.php

```diff
@@ -656,65 +656,74 @@ function getAncestorsOf($table, $items_id) {
    $parentIDfield = getForeignKeyFieldForTable($table);
    $use_cache     = FieldExists($table, "ancestors_cache");
 
-   if ($use_cache
-       && ($items_id > 0)) {
+   if(!is_array($items_id)){
+      $items_id = array($items_id);
```

Comment:
You no longer check if the item is greater than 0. I guess array should be filtered to drop all those cases.